lucien-ios/Protocols/DismissViewController.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  DismissViewController.swift
+//  lucien-ios
+//
+//  Created by Fang, Gracie on 10/20/17.
+//  Copyright © 2017 Intrepid Pursuits. All rights reserved.
+//
+
+import Foundation
+
+protocol DismissViewController {
+
+    func dismissPresentedViewController() 
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Trailing Whitespace Violation: Lines should not have trailing whitespace. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.intrepid.io/coding_rules#rule_key=SwiftLint%3Atrailing_whitespace)